src/Directory.Build.props

```diff
@@ -13,6 +13,11 @@
     <AnalysisLevel>latest</AnalysisLevel>
     <AnalysisMode>recommended</AnalysisMode>
     <SharedRoot>$(SrcRoot)Shared/</SharedRoot>
+    <!-- This setting makes symbol files (PDBs) available within the DLLs
+    themselves, which is seemingly necessary for having a 'deterministic' build checkmark in our NuGet package.
+    This is recommended for NuGet packages, as per: https://devblogs.microsoft.com/dotnet/producing-packages-with-source-link/
+    The connection to deterministic builds is explored here: https://www.meziantou.net/creating-reproducible-build-in-dotnet.htm -->
+    <DebugType>embedded</DebugType>
```

Comment:
I recall that Jacob had preferred non-embedded symbols in our nuget package, and instead wanted us to publish symbol packages separately (snupkg files). Unfortunately, I don't have experience working with snupkg files, but I suspect the DebugType value for this is `Portable`, based on the metadata I see in previous versions of these packages.

I don't think we need to block on this, but it might be worth taking a look to see whether this might impact the release process, if for example it expects to find snupkg files but none are generated (because we use embedded symbols).